apollo/helpers.ts

```diff
@@ -5,7 +5,37 @@ import {
   Talent,
   Organization,
   TalentUpdate,
+  Experience,
+  TalentAssetEntry,
+  BaseEntity,
+  ExperienceEntry,
+  TalentEntry,
 } from '../lib/types';
+import { nanoid } from 'nanoid';
+
+const handleError = (e: Error): Error => {
+  console.error(e); //eslint-disable-line no-console
+  return e;
+};
+
+//eslint-disable-next-line @typescript-eslint/ban-types
+const updateObject = (object: object, update: object | undefined): object => {
+  if (!update) return object;
+  return { ...object, ...update };
+};
+
+const getExperienceById = (
+  talent: TalentEntry | undefined,
+  id: string,
+): ExperienceEntry | null => {
+  if (!talent) return null;
+  if (!talent.experiences.length) return null;
+  const experiences = talent.experiences.filter(
+    (experience) => experience.id === id,
+  );
+  if (!experiences.length) return null;
+  return experiences[0];
+};
 
```

Comment:
can we not use the "filterById" in utils/arrays ?
I thought that you created it for use cases like this

Reply:
that's right. I added it after I wrote getExperienceById. With last push I'm using filterById within getExperienceById, since the latter does checks and returns null instead of undefined if they fail. Might be a good idea to think a bit harder about where we want to return undefined instead of null or vice versa.